### PR TITLE
basic func of line count for lmr

### PR DESCRIPTION
**Pull request by aboft**

### CODE REVIEW

**Comment by aboft** [comment]
image

File: frontend/components/Dashboard.tsx [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
+		    </Col>
+		    <Col md>
+			<LineCountList/>
+		    </Col>
+		</Row>
+		<Row >
+                    <Col md >
                         <MessageList></MessageList>
                     </Col>
                 </Row>
```
Comment (LoLei) [inline frontend/components/Dashboard.tsx]: WTF is this indentation lmao

File: frontend/components/LineCountList.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    return (
+        <div>
+            <div className={"messagelist-header"}>
+                Current Line Count:
+            </div>
+            {fetchedLines.map((line) =>
+                <LineCount key={line.dateCreated || Date.now()}
+                    count={line.count}
+                    dateCreated={line.dateCreated}
+		    message={line.message}
```
Comment (LoLei) [inline frontend/components/LineCountList.tsx]: Fix eyesore indentation pls

**Review by LoLei — CHANGES_REQUESTED** [review]
Functionally 👍

**Review by LoLei — APPROVED** [review]

**Pull request merged**